processing/src/main/java/io/druid/segment/StorageAdapter.java

```diff
@@ -53,16 +53,16 @@
   Comparable getMinValue(String column);
   @Nullable
   Comparable getMaxValue(String column);
-  Capabilities getCapabilities();
+  boolean areDimensionValuesSorted();
```

Comment:
is this used in extensions anywhere?

Reply:
Good question, I don't know the answer. Anyone?

Reply:
I can _not_ remove it if preferable, since class is marked `@PublicApi`, I wasn't paying close attention.

Reply:
This can be removed, but if so, this change can be applied to only major releases.

Reply:
Bleh, yeah. I will back out this change and split removal of this class into another PR